src/main/java/org/sonar/plugins/buildbreaker/QualityGateBreaker.java

```diff
@@ -110,6 +113,16 @@ private void execute() {
     checkQualityGate(wsClient, analysisId);
   }
 
+  private String getServerUrl(Properties reportTaskProps) {
+	  String altServerUrl = settings.getString(BuildBreakerPlugin.ALTERNATIVE_SERVER_URL_KEY);
+	  if (isNullOrEmpty(altServerUrl)) {
+		  return reportTaskProps.getProperty("serverUrl");
+	  } else {
+		  LOGGER.debug("Using alternative server url ({}): {}", BuildBreakerPlugin.ALTERNATIVE_SERVER_URL_KEY, altServerUrl);
+		  return altServerUrl;
+	  }
```

Comment:
Please keep indentation consistent with the rest of the file.  Use indentation of 2 spaces.  Do not use tabs.